app/webroot/js/sentences_lists.menu.js

```diff
@@ -18,127 +18,112 @@
 
 
 $(document).ready(function() {
+    // Clicking on "Add to list" displays the list selection.
+    // Reclicking on it hides the list selection.
+    $(".addToList").click(function(){
+        sentenceId = $(this).attr("data-sentence-id");
+        $("#addToList"+sentenceId).toggle();
+    });    
+
+    // The sentence is added to the list after user has clicked
+    // the button. I was hesitating between this solution and
+    // having the sentence added to the list onChange, that is,
+    // directly after the selection in the <select>.
+    $(".validateButton").click(function(){

```

Comment:
Had the above .addToList worked this would've resulted in two calls to add_sentence_to_list.